content/_code-samples/escrow/js/get-escrows.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+if (typeof module !== "undefined") {
+  // Use var here because const/let are block-scoped to the if statement.
+  var xrpl = require('xrpl');
+};
+
+const secret = "sEdTPPEeMH6SAgpo6rSj8YW7a9vFfUj";
+const source = ""; // Optional: filter by source
+const destination = ""; // Optional: filter by destination
```

Comment:
nit: Since the user would be entering in an address that they intend on searching, we should be able to simply filter the escrow objects using `address`

Reply:
My intention was that giving an option will be a good, if someone wants to filter using a different source/destination

Reply:
It's a bit unclear what these parameters actually should be. It might be useful to give them names that imply their type like `sourceAddress`. 

Also, it'd be good to add a log in case someone doesn't specify either -Ex. Printing "You can filter results to just display those with a specific source address or destination address by modifying `sourceAddress` or `destinationAddress` in this code sample." at the end if neither were specified.

Reply:
Added